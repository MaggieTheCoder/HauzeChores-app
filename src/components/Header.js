import React from "react";
import "../styles/Header.css";

const Header = () => {
  return (
    <header className="header">
      <div className="header__banner">banners</div>
      <h1 className="header__h1">Houze</h1>
      <h3 className="header__h3">Just get it done!</h3>
    </header>
  );
};

export default Header;
