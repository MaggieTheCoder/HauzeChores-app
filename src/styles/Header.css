.header {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.header h1 {
  font-size: 160px;
  font-weight: 1300px;
  font-style:bold;
  margin-bottom: 30px;
  margin-top: 180px;
  letter-spacing: -2px;
  color: #C60982;
  -webkit-text-stroke: 0.3px#360866;
  text-shadow:
      3px 3px 0 #000,
    -2px -1px 0 #000,  
     1px -1px 0 #000,
     -1px 1px 0 #000,
      1px 1px 0 #000;
}
.header h3 {
  margin-top: 20px;
  margin-bottom: 20px;
  font-size : 200%;
  color :#360866;
}

.header__banner {
  background : #C60982;
  color: #C60982;
	position : absolute ;
	left : 0px;
	top : 0px;
	width : 100%;
	height : 100px;
	border-radius : 0px 0px 95px 95px;
	-moz-border-radius : 0px 0px 106px 106px;
	-webkit-border-radius : 0px 0px 106px 106px;
}
