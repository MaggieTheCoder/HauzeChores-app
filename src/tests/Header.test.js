import React from "react";
import { render } from "@testing-library/react";
import Header from "../components/Header";

test("renders app correctly", () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});
