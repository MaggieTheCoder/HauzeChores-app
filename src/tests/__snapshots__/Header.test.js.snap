// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app correctly 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <div
      class="header__banner"
    >
      banners
    </div>
    <h1
      class="header__h1"
    >
      HOUZE
    </h1>
    <h3
      class="header__h3"
    >
      Just get it done!
    </h3>
  </header>
</DocumentFragment>
`;
